pkg/elb/cleanup.go

```diff
@@ -199,6 +199,24 @@ func getServiceLoadBalancer(ctx context.Context, ec2API awsapi.EC2, elbAPI Descr
 	return &lb, nil
 }
 
+func getIngressELBName(ctx context.Context, hosts []string) (string, error) {
```

Comment:
We can remove `ctx context.Context` from the params

Reply:
Agree, I removed the unused `ctx` parameter.